cases/models.py

```diff
@@ -125,8 +127,13 @@ def get_merged_into_cases(self, cases):
 
         merge_map = {c.id: [] for c in cases}
         for action in query:
+            # Even though merging actions should not have an edited 'time', we need to
+            # use 'time' rather than 'created' for the merged 'at'.
+            # This is because in other action queries we use 'time' and we should be consistent,
+            # especially as there is a small delta between the values of 'time' and 'created' for
+            # a newly created action.
```

Comment:
Fine to be consistent, yes, just curious if you did find an issue using created?